### ISSUE TITLE
Extended functionality for color components.

### ISSUE THREAD

**Issue by RyanHavoc** [NoDivide/astrum#92]
Improve functionality of color components. Discussions include: #25, #16, #35

**Comment by JonasEriksson** [comment]
My vote for adding this as well, fine enough to manually add it into the colors section right now if possible (how?). It seems very clear that variable names, using SCSS etc., are very important to a styleguide/pattern-library, not just the finished hex value. Is this coming at all, given it's been a year? Thanks for considering it!

**Comment by RyanHavoc** [comment]
Yes, definitely something I'll be adding, but it's part of a wider overhaul I'm planning for Astrum. Backward compatibility but practically a rewrite from the ground up with lots of new features (including this!).
